ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -160,6 +160,7 @@ ANALYTICS_IDS = {
     'GOOGLE_ANALYTICS_ID': '{{ localsettings.GOOGLE_ANALYTICS_ID }}',
     'PINGDOM_ID': '{{ localsettings.PINGDOM_ID }}',
     'SEGMENT_ANALYTICS_KEY': '{{ localsettings.SEGMENT_ANALYTICS_KEY }}',
+    'HUBSPOT_ID': '{{ localsettings.HUBSPOT_ID }}',
```

Comment:
This will be `''` or `'None'` (or something else?) if `localsettings.HUBSPOT_ID` isn't defined